content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,281 @@
+
+They are mainly responsible for:
+
+- keeping up to date the NGI entry in the Configuration Database and managing
+  the status of all sites under their NGI, and ensuring their information is
+  also kept current
+- addressing problems with Site availability or reliability. The reports are
+  issued on a monthly basis and the NGI operations managers have 10 days to
+  respond to identified problems
+- attending regular
+  [Operations Management Board meetings](https://indico.egi.eu/category/19/)
+
+All NGI operations management responsibilities are listed in the
+[Resource Infrastructure Provider OLA document](https://documents.egi.eu/document/463).
+
+### Project level
+
+#### Chief Operations Officer
+
+The Chief Operations Officer leads EGI Operations, and is responsible for
+coordinating the operations of the infrastructure across the project.
+
+#### EGI CSIRT
+
+[EGI CSIRT](https://csirt.egi.eu) is the official security coordination team and
+contact point at project level.
+
+##### EGI IRTF
+
+The role of EGI CSIRT's Incident Response Team for the Federation (IRTF) team is
+to handle day to day operational security issues and coordinate _Computer
+Security Incident Response_ across the EGI infrastructure.
+
+#### EGI CSIRT Officer
+
+The member of EGI Incident Response Team for the Federation (IRTF) currently on
+shift.
+
+#### EGI Foundation SDIS team
+
+The [EGI Foundation](https://ims.egi.eu/display/EGIG/EGI+Foundation) Service
+Delivery and Information Security (SDIS) team, formerly known as the EGI
+Operations team, is responsible of
+[coordinating and supporting the operational activities](https://confluence.egi.eu/display/EGIPP/EGI+Infrastructure+operations+oversight)
+of all the EGI Infrastructure.
+
+#### VO
+
+A
+[Virtual Organisation (VO)](https://ims.egi.eu/display/EGIG/Virtual+organisation)
+is a group of users and, optionally, resources, often not bound to a single
+institution or national borders, who, by reason of their common membership and
+in sharing a common goal, are given authority to use a set of resources. Each VO
+member signs the VO Acceptable Usage Policy (AUP) (during registration) which is
+the policy describing the goals of the VO thereby defining the expected and
+acceptable use of the resources by the users of the VO. User documentation can
+be found [in the users section](../../../users/) and
+[in the EGI Wiki](https://wiki.egi.eu/wiki/User_Documentation).
+
+#### VO manager
+
+An individual responsible for the management of the membership registry of the
+VO ensuring its accuracy and integrity.
+
+## Joining operations
+
+In order to join any of the organisational groups in your NGI, you will need to
+go through the following steps in order:
+
+### Authentication
+
+The authentication in EGI infrastructure works either with X509 personal
+certificates or through federated identities using [Check-in](../../check-in).
```

Comment:
@paolini78 maybe we should mention that we are in a transition from X509 to federated identities with tokens/ assertions via OpenID Connect/SAML. Seeing this and the following sections it looks like X509 is the main authentication method (at least as it's mentioned first).

Reply:
I will try to rephrase saying that some servuices are only x509 compliant (I didn't want to go too much in detail...)

Reply:
I have updated the text.